### ISSUE TITLE
modding menu/api

### ISSUE THREAD

**Issue by oneyusof** [Carroted/Simulo#26]
basically its quite simple actually, there will be a menu for mods on the top of your screen (next to the File menu and the Settings menu)

the modding API would be fairly simpler, although I kind of don't see a point of the modding API after all as the client controller is already defined globally so anyone has access, which means the `clientController` object can do as much as the API could possible do, you could spawn objects and such.

# Mod Menu
It would look like a cool menu where you could browse and enable mods, disable mods and even edit mods ingame using the monaco editor

# Context

this is just my idea of how modding in simulo would work but right now you could probably think of something else

**Comment by aMySour** [comment]
yeah I already said something similar with Monaco in-game scripting menu

anyway I was gonna make this right after multiplayer which is why no issue

the menu for it will be in main menu

**Comment by oneyusof** [comment]
# ok nerd
